lib/active_record/connection_adapters/amazon_timestream_adapter.rb

```diff
@@ -14,9 +14,8 @@ def self.amazon_timestream_connection(config)
       config = config.symbolize_keys
 
       raise ArgumentError, 'No database specified. Missing argument: database.' unless config.key?(:database)
-
-      credentials = Aws::Credentials.new config[:username], config[:password] if config[:username] && config[:password]
-      connection = Aws::TimestreamQuery::Client.new({ credentials: credentials }.compact)
+      credentials = Aws::Credentials.new config[:username], config[:password], config[:session_token] if config[:username] && config[:password]
```

Comment:
This line is getting long, can we replace `config[:username], config[:password], config[:session_token]` with `config.values_at(:username, :password, :session_token)`?